components/prize_pool/commons/prize_pool.lua

```diff
@@ -565,10 +566,21 @@ function PrizePool:_buildRows()
 			end
 
 			if opponentIsParty then
-				if opponent.opponentData.players[1] and opponent.opponentData.players[1].team then
-					row:addCell(TableCell{content = {tostring(OpponentDisplay.BlockOpponent{
-						opponent = {type = 'team', template = opponent.opponentData.players[1].team}
-					})}, css = opponentCss})
+				local playerTeams = mw.html.create('div')
+				local hasTeam
+				for _, player in ipairs(opponent.opponentData.players) do
+					if player.team then
+						hasTeam = true
+						playerTeams:node(OpponentDisplay.BlockOpponent{
+							opponent = {type = 'team', template = player.team}
+						})
+					else
+						playerTeams:node(EMPTY_TEAM)
+					end
+				end
+
+				if hasTeam then
+					row:addCell(TableCell{content = {tostring(playerTeams)}, css = opponentCss})
```

Comment:
```suggestion
				local playerTeams = {}
				for _, player in ipairs(opponent.opponentData.players) do
					table.insert(playerTeams, tostring(OpponentDisplay.BlockOpponent{
						opponent = {type = 'team', template = player.team}
					}))
				end

				if #playerTeams > 0 then
					row:addCell(TableCell{content = playerTeams, css = opponentCss})
```
Would this work?

Reply:
nope
the additional div wrapper is needed so that they are shown below each other instead of besides each other

first opponent without the div wrapper, 2nd with the wrapper:
![Screenshot 2022-08-05 10 13 02](https://user-images.githubusercontent.com/75081997/183033406-3445624d-ec15-465a-8b33-6d6412f55229.png)

Reply:
```suggestion
				local playerTeams = {}
				for _, player in ipairs(opponent.opponentData.players) do
					table.insert(playerTeams, {tostring(OpponentDisplay.BlockOpponent{
						opponent = {type = 'team', template = player.team}
					}}))
				end

				if #playerTeams > 0 then
					row:addCell(TableCell{content = playerTeams, css = opponentCss})
```
Try this

Reply:
still besides eachother
(plus line 573 should be `})})`, not `}}))`)

Reply:
```suggestion
				local playerTeams = {}
				for _, player in ipairs(opponent.opponentData.players) do
					if player.team then
						table.insert(playerTeams, {tostring(OpponentDisplay.BlockOpponent{
							opponent = {type = 'team', template = player.team}
						})})
					else
						table.insert(playerTeams, {DASH})
					end
				end

				row:addCell(TableCell{content = playerTeams, css = opponentCss})
```
?

Reply:
1) still besides each other due to the missing div
2) still would show several dashes in the same cell if none of them has a team
3) the dash is now left aligned (which is why i added the div with `:css('text-align', 'center')` around it in the first place

with my code suggestion:
![FA888C7B-319F-443C-9210-84C9B655247C](https://user-images.githubusercontent.com/75081997/183272069-b82b9e80-03fa-49db-9fe8-d17ff893b962.jpeg)

with your above suggestion:
![E6B3DD05-4F29-4946-AE33-EAE7909516EB](https://user-images.githubusercontent.com/75081997/183272070-4a38544b-6a5f-4696-9cf1-98c533e95d4e.jpeg)